docs/data/material/components/timeline/timeline.md

```diff
@@ -54,3 +54,21 @@ Here is an example of customizing the component.
 You can learn more about this in the [overrides documentation page](/material-ui/customization/how-to-customize/).
 
 {{"demo": "CustomizedTimeline.js"}}
+
+## Alignment
+
+There are different ways in which a Timeline can be placed within the container. You can do it by overriding the styles.
```

Comment:
```suggestion
There are different ways in which a Timeline can be placed within the container. 
You can do it by overriding the styles.
```

Nit-picky to have each sentence in a single line 😬

Reply:
I am a bit skeptical to always have each sentence in a new line. It sometimes loses the information connection/continuation for the reader, IMO. But anyway, I have done it as you suggested.

Reply:
@ZeeshanTamboli Markdown doesn't register it as a new line unless you add an extra line break in between, as in the suggestion below (I'm not actually suggesting this, just showing what it would look like). 

Giving each sentence its own line makes it much easier for us to offer line-by-line suggestions, just like with code. :)

```suggestion
There are different ways in which a Timeline can be placed within the container. 

You can do it by overriding the styles.
```